plugins/device/deviceutils/utils.go

```diff
@@ -12,8 +12,16 @@ import (
 )
 
 const (
-	maxMetadataSizeGb = 16
-	chunkSize         = "1280k"
+	KB              = 1000
```

Comment:
`ValidateSizeList` function in `glusterd2/options/options.go` can be updated with these values